src/blockchain/sync-augur-node-with-blockchain.ts

```diff
@@ -5,23 +5,23 @@ import { startAugurListeners } from "./start-augur-listeners";
 import { downloadAugurLogs } from "./download-augur-logs";
 
 interface HighestBlockNumberRow {
-  highest_block_number: number
+  highest_block_number: number;
 }
 
 export function syncAugurNodeWithBlockchain(db: Knex,  augur: Augur, ethereumNodeEndpoints: EthereumNodeEndpoints, uploadBlockNumbers: UploadBlockNumbers, callback: ErrorCallback): void {
   augur.connect({ ethereumNode: ethereumNodeEndpoints }, () => {
     console.log("Waiting for first block...");
     startAugurListeners(db, augur, (blockNumber: string) => {
       db.raw(`SELECT highest_block_number FROM blockchain_sync_history ORDER BY highest_block_number DESC LIMIT 1`)
-        .asCallback( (err: Error|null, rows?: HighestBlockNumberRow[]) => {
-          if(err) return callback(err);
+        .asCallback( (err: Error|null, rows?: Array<HighestBlockNumberRow>) => {
```

Comment:
I believe it's best practice to explicitly include a `void` return type here, i.e.:
```
(err: Error|null, rows?: Array<HighestBlockNumberRow>): void => {
```

Reply:
Note: the `: void` return type is actually missing in a number of places / I'm not so good at remembering to include this myself.

Reply:
I'll add the lint rule for that and see how bad the damage is

Reply:
Quite a few places that don't do the return types.

Mind if we merge this and I do a separate pass for these so I don't get into conflicts?